src/Illuminate/Foundation/Http/FormRequest.php

```diff
@@ -165,6 +165,7 @@ protected function failedValidation(Validator $validator)
     {
         $exception = $validator->getException();
 
+        /** @var \Illuminate\Validation\ValidationException $exception */
```

Comment:
How is `$exception` is not a string?

The return type of `Illuminate\Validation\Validator@getException()` is `class-string<\Illuminate\Validation\ValidationException>`, so `$exception` is a string.

https://github.com/laravel/framework/blob/61492a8846a1ab4bd6ab01e3edf90007ba818753/src/Illuminate/Validation/Validator.php#L1535-L1543

Furthermore, I don't think the `new` operator can be used with anything other than a string.